apps/editing-toolkit/editing-toolkit-plugin/starter-page-templates/class-starter-page-templates.php

```diff
@@ -277,17 +290,27 @@ public function enqueue_assets() {
 	public function get_page_templates( string $locale ) {
 		$page_template_data   = get_transient( $this->get_templates_cache_key( $locale ) );
 		$override_source_site = apply_filters( 'a8c_override_patterns_source_site', false );
+		$is_assembler_v2_site = 'pub/assembler' === get_stylesheet();
```

Comment:
I haven't checked but I'm pretty sure in a Jetpack site, the stylesheet is `assembler`, not `pub/assembler`. I guess it is OK for now; we can just ask the tester to test with Simple sites only.

Reply:
Good catch! I have pushed a fix and tested it on Atomic.

<img width="1296" alt="Screenshot 2567-01-12 at 14 55 41" src="https://github.com/Automattic/wp-calypso/assets/1881481/9c9b3bc5-32a4-4940-8846-9981ef426358">